airflow/hooks/hive_hooks.py

```diff
@@ -46,29 +52,117 @@ class HiveCliHook(BaseHook):
 
     The extra connection parameter ``auth`` gets passed as in the ``jdbc``
     connection string as is.
+
+    :param mapred_queue: queue used by the Hadoop CapacityScheduler
```

Comment:
Actually, I believe that this part works with the fair scheduler as well. Just double checked the documentation. That said the parameters as is are targeting Hadoop 2 and I guess we should be conscious of that.